src/main/java/net/minecraftforge/fml/client/GuiMultipleModsErrored.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.fml.client;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.ScaledResolution;
+import net.minecraft.client.resources.I18n;
+import net.minecraftforge.fml.common.*;
+
+import java.util.List;
+
+public class GuiMultipleModsErrored extends GuiErrorBase
+{
+    private final List<RuntimeException> exceptions;
+    private int pageIndex;
+    private GuiErrorBase subGui;
+
+    public GuiMultipleModsErrored(MultipleModsErrored exception)
+    {
+        super();
+        this.exceptions = exception.exceptions;
+    }
+
+    private void switchSubGui()
+    {
+        RuntimeException exception = exceptions.get(pageIndex);
+        if (exception instanceof WrongMinecraftVersionException)
+        {
+            subGui = new GuiWrongMinecraft((WrongMinecraftVersionException) exception);
+        }
+        else if (exception instanceof MissingModsException)
+        {
+            subGui = new GuiModsMissing((MissingModsException) exception);
+        }
```

Comment:
Why make these different 'pages', why not just make a scroll window and list all of them at once, this would be better.